services/gitclub/app/fixtures.py

```diff
@@ -13,7 +13,38 @@
 FAKE_REPOSITORIES = 20
 FAKE_ISSUES = 100
 
+def limit_bulk_tell(facts, bulk_limit=20):
+    start_index = 0
+    end_index = min(len(facts), bulk_limit)
+    total_facts_added = 0
+    total_number_facts = len(facts)
+    while start_index < total_number_facts:
+        oso_response =oso.bulk_tell(facts=facts[start_index:end_index])
+        if oso_response != None:
```

Comment:
It looks like if `oso_response` is anything other than `None`, an error occurred, and the code will `print("An issue occurred...")`. Otherwise, `total_facts_added` will be incremented.

In the failure case (when `oso_response != None`), I think this code will continue and retry, in the case that an error did occur  - is that intended? If so, should we put a limit on the number of retries? I'm not familiar with this codebase, so this is much more of a question than a suggestion :)

Reply:
Also, given that the code before this change was `print(oso.bulk_tell(facts=facts))`, then `oso_response` seems as though it might provide a string with some error output. Is that true? If so, could we show the user this error message? (I didn't read the definition of `oso.bulk_tell` yet)

Reply:
The loop should continue updating the list and try pushing "other" facts from the list. It doesn't fail the entire upload of facts if one bulk_tell fails. Not sure what the best action is here. Currently it will print that an error occurred and the range of facts where the error occurred.

It would be good to also print the error message (if there is one). I also did not look too closely at what is returned. I'll follow up on that. It would be more helpful for the user and Oso Engineers down stream helping with support.